templates/patterns/card/pattern-card.html.twig

```diff
@@ -5,51 +5,52 @@
  */
 #}
 {% spaceless %}
-{% if title is not empty %}
-  {% set title = {
-    'content': title,
-  } %}
+{% set content %}
+  {% if meta is not empty %}
+    {# Use negative margin to compensate for me-3 on each item. #}
+    <div class="my-3 me-n3">
+      {% for _meta_item in meta %}
+        <span class="text-muted me-3">{{ _meta_item }}</span>
+      {% endfor %}
+    </div>
+  {% endif %}
+  <div>{{ content }}</div>
+{% endset %}
+{% if url is not empty %}
+  {% set title %}
+    <a href="{{ url }}" class="text-underline-hover">{{ title }}</a>
+  {% endset %}
 {% endif %}
-{% if subtitle is not empty %}
-  {% set subtitle = {
+{% include '@oe-bcl/card' with {
+  'horizontal': orientation == 'horizontal',
+  'horizontal_grid': (orientation == 'horizontal') ? {
+    left_col_classes: 'col-4',
+    right_col_classes: 'col-8',
+    gutter: 0,
+  } : {},
+  'title': (title is not empty) ? {
+    'content': title,
+  } : {},
+  'subtitle': (subtitle is not empty) ? {
     'content': subtitle,
     'classes': 'mb-2',
-  } %}
-{% endif %}
-{% if text is not empty %}
-  {% set text = {
+  } : {},
+  'text': (text is not empty) ? {
     'content': text,
     'classes': 'mb-2',
-    'tag': 'div'
-  } %}
-{% endif %}
-{% if orientation == 'horizontal' %}
-  {% set horizontal_grid = {
-    left_col_classes: 'col-4',
-    right_col_classes: 'col-8',
-    gutter: 0
-  } %}
-{% endif %}
-{% if date is not empty %}
-  {% set _date = {
-    'day': date|date('d'),
-    'month': date|date('M'),
-    'year': date|date('Y'),
-    'date_time': date|date('Y-m-d')
-  } %}
-{% endif %}
-{% include '@oe-bcl/card' with {
-  'horizontal': orientation == 'horizontal',
-  'horizontal_grid': horizontal_grid,
-  'title': title,
-  'subtitle': subtitle,
-  'text': text,
+    'tag': 'div',
+  } : {},
   'content': content,
   'image': image,
   'card_header': header,
   'card_footer': footer,
   'badges': badges,
-  'date': _date,
+  'date': (date is not empty) ? {
+    'day': date|date('d'),
+    'month': date|date('M'),
+    'year': date|date('Y'),
+    'date_time': date|date('Y-m-d'),
+  } : {},
```

Comment:
We can remove date until full date ticket is prepared.

Reply:
I notice that date is not even declared in the pattern definition.
I kept it there because it existed in the current version of the template.
But I agree to remove it.